browser/static/ably.js

```diff
@@ -7693,6 +7703,42 @@ var RealtimeChannel = (function() {
 		this.pendingEvents = [];
 	};
 
+	RealtimeChannel.prototype.history = function(params, callback) {
```

Comment:
It's impossible to see what changes are here because your editor is changing the whitespace in the file.